front/cypress/e2e/admin/pages_and_menu/update_homepage_hero_banner_content.cy.ts

```diff
@@ -164,26 +147,22 @@ describe('Admin: update Hero Banner content', () => {
     cy.get('.e2e-submit-wrapper-button').click();
     cy.wait('@saveHomePage');
     cy.get('.e2e-submit-wrapper-button').contains('Success');
-  });
 
-  it('views the signed-in content properly as a verified user', () => {
-    cy.setLoginCookie(verifiedEmail, verifiedPassword);
+    // log out, view page as verified user
```

Comment:
Where do we log out and view the page as a verified user here? We just go to the landing page, right?

Btw, shall we rename goToLandingPage and e2e-landing-page to homepage equivalents now that we renamed the component?

Reply:
Comments have been clarified